src/services/config/index.ts

```diff
@@ -65,7 +66,7 @@ class ConfigService {
 
   private readonly userConfigMustExist = (): void => {
     this.configFileMustExist()
-    const userId = this.getCurrentUser()
+    const userId = this.getCurrentUser() || vaultService.getSession().account.userId
```

Comment:
`vaultService.getSession().account.userId`
are we sure that we don't need optional chaining here to prevent "cannot get property of undefined" issue?